### DISCUSSION TITLE
show Resource page title without action prefix (i.e. for view page only show "<record title>", not "View <record title>")

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#8108] [Help] [answered]
For a page (e.g. View) it defaults to showing either static title or dynamic title (`<action>: <record title>`).

```
# ViewRecord.php

    public function getTitle(): string | Htmlable
    {
        if (filled(static::$title)) {
            return static::$title;
        }

        return __('filament-panels::resources/pages/view-record.title', [
            'label' => $this->getRecordTitle(),
        ]);
    }
```

Can I just dynamically show the record title but without the action prefix?
So show title: `John Doe`  
Instead of: `View John Doe`

**Comment by danharrin** [comment, marked as answer]
Yup, override `getTitle()` to just return `$this->getRecordTitle()`

**Comment by wivaku** [comment]
Thanks!  
Ah, I see this is [mentioned in the documentation](https://filamentphp.com/docs/3.x/panels/pages#customizing-the-page-title) (using static title).  
I did not connect the dots you can use the same `$this-> getRecordTitle()` instead of a static title.

Full example:
```
# Resources\PostResource\Pages\ViewRecord.php

    public function getTitle(): string | Htmlable
    {
        return $this->getRecordTitle();
    }
```
